.pre-commit-config.yaml

```diff
@@ -30,6 +30,16 @@ repos:
           - prettier@3.2.5
```

Comment:
temporary comment 'type_or' line and add a comment for that: 
"TODO: Uncomment this line when Astro file support is available"